src/content/en/updates/2015/07/interact-with-ble-devices-on-the-web.md

```diff
@@ -372,6 +390,7 @@ Check out our [curated Web Bluetooth Demos](https://github.com/WebBluetoothCG/de
 ## Libraries
 
 - [web-bluetooth-utils](https://www.npmjs.com/package/web-bluetooth-utils) is a npm module that adds some convenience functions to the API.
+- A Web Bluetooth API shim is available in [noble](https://github.com/sandeepmistry/noble), the most popular Node.js BLE central module. This allows you to webpack/browserify noble without the need for a WebSocket server or other plugins.
```

Comment:
So this is a webpack'able library is what I'm reading. Correct me if I'm wrong (sorry a bit late to this party [but part of the webpack team and wanted to make sure we catch any related questions in our github if they come up])